python/mcap-ros2-support/README.md

```diff
@@ -0,0 +1,26 @@
+# Python MCAP ROS2 support
+
+This package provides ROS2 support for the Python MCAP file format reader.
```

Comment:
More detail – maybe mention this provides support for ROS2 when _not_ working in a ROS 2 environment? Or does it support both (and if so, do you get any additional features by being in a ROS environment)?